coding-concepts.js

```diff
@@ -10,37 +10,46 @@
 var cohort = "Bravo 2022"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "B" "r" "a" "v" "o"  "2" "0" "2" "2"
+// b) Verify and explain: The answer is an array of the elements broken up. The split method takes the variable contents and splits each index into an individual element. The output comes back as an array to contain all new elements.
+//Output:
+// [
+//   'B', 'r', 'a', 'v',
+//   'o', ' ', '2', '0',
+//   '2', '2'
+// ]
 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
-  `Hello, ${name}!`
+ `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
-
-// a) Your answer:
-// b) Verify and explain:
+console.log(greeter("LEARN Student"))
 
+// a) Your answer: `Hello, LEARN Student`
+// b) Verify and explain: The answer came back as undefined because the function is missing a return before the string. Without the return the parameter isn't acted on to give an output.
+//Output:
+// undefined
 
 // --------------------3) What will this log?
 
 var multBy2 = [4, 5, 6, 7, 8].map(value => value * 2)
-// console.log(multBy2)
+console.log(multBy2)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The answer will be undefined
+// b) Verify and explain: The variable can have the Higher Order Function in the variable declaration. Then in the console.log the variable is acted on, running through the .map method included in its description.
```

Comment:
For sure!  A higher-order function does not need to be inside a custom function to be utilized.  In this case, the return array will be assigned to the variable multBy2.